AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/CustomEndpoint/AWSEndpointResolving.swift

```diff
@@ -19,8 +24,8 @@ struct AWSEndpointResolving: AWSClientRuntime.EndpointResolver {
         self.endpoint = endpoint
     }
 
-    func resolve(serviceId: String, region: String) throws -> AWSEndpoint {
-        let endpoint = try endpoint()
-        return AWSEndpoint(endpoint: endpoint)
-    }
+//    func resolve(serviceId: String, region: String) throws -> AWSEndpoint {
```

Comment:
Should we remove the commented out code?

Reply:
Addressed in [5514065](https://github.com/aws-amplify/amplify-swift/pull/2552/commits/5514065c358ab5fa987c9fd1513bed3d1ffef563)